pom.xml

```diff
@@ -169,6 +166,14 @@
     </contributor>
   </contributors>
 
+  <licenses>
+    <license>
+      <name>Apache License 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
   <scm>
     <connection>scm:git:https://github.com/mathieucarbou/${mycila.github.name}.git</connection>
     <developerConnection>scm:git:git@github.com:mycila/${mycila.github.name}.git</developerConnection>
```

Comment:
placeholders could be replaced here

Reply:
ok will take care of cleaning up some of that, I was thinking the same when doing this :)